go-controller/pkg/cluster/node.go

```diff
@@ -74,59 +74,65 @@ func (cluster *OvnClusterController) StartClusterNode(name string) error {
 	var subnet *net.IPNet
 	var clusterSubnets []string
 	var cidr string
-	var connected bool
+	var readyChan = make(chan bool, 1)
+
+	err = cluster.watchConfigEndpoints(readyChan)
```

Comment:
@alexanderConstantinescu if `config.OvnNorth.Address` and `config.OvnSouth.Address` are already set, then we don't need to block waiting for the `readyChan` because the user explicitly set the DBs on the command/line config and we let them shoot themselves in the foot.

But if the address is empty, then yes we should wait.